airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -205,17 +205,15 @@ def post(
         response = self._session.post(self.BASE_URL + url, params=params, json=data)
         return self._parse_and_handle_errors(response), response
 
-    def get_custom_object_schemas(self) -> Mapping[str, Any]:
+    def get_custom_objects_metadata(self) -> Tuple[str, str, Mapping[str, Any]]:
```

Comment:
I believe the return type here is not right. It should be `Iterable[Tuple[str, str, Mapping[str, Any]]]` since we are returning the generator that returns each tuple.

Reply:
thanks, fixed